grails-app/services/icaruswings/PayerService.groovy

```diff
@@ -81,6 +81,22 @@ class PayerService {
         payer.save(failOnError: true)
     }
 
+    public void deleteAllPayersForCustomer(Long customerId) {
+        List<Long> payerIds = PayerRepository.query([ customer: customerId ]).column("id").readOnly().list()
+
+        if (!payerIds.isEmpty()) {
```

Comment:
precisa? Se chegar no for e não tiver registros ele só não vai entrar nunca, certo?